### DISCUSSION TITLE
feature: sortable bank

### DISCUSSION THREAD

**Discussion by Daywalkrr** [AscensionGameDev/Intersect-Engine#2408] [Ideas]
**Is your feature request related to a problem? Please describe.**
The bank, in-game, can get unwieldy, fast.

**Describe the solution you'd like**
It would be nice if we were able to sort the bank in some way. Perhaps there could be a drop down of sort options, and a sort button the bank, that allows us to do just that. Would also be nice if this extended to the inventory.

**Comment by Cheshire92** [comment]
As an fyi, if you want to PR this I will **NOT** be merging anything bank related until after https://github.com/AscensionGameDev/Intersect-Engine/pull/959 gets merged.

I really don't want to deal with bank related merge nightmares after editing that in until I refactor that.

**Comment by Daywalkrr** [comment]
Go ahead and ping me when 959 is merged, and I'll resolve conflicts with my stuff before hand.
